pkg/purge_osd/purge_osd.go

```diff
@@ -0,0 +1,33 @@
+package purgeosd
+
+import (
+	"context"
+	"fmt"
+
+	cephcommands "github.com/red-hat-storage/odf-cli/pkg/ceph"
+	"github.com/rook/kubectl-rook-ceph/pkg/k8sutil"
+	"github.com/rook/kubectl-rook-ceph/pkg/logging"
+	"github.com/rook/kubectl-rook-ceph/pkg/rook"
+)
+
+func PurgeOsd(ctx context.Context, clientsets *k8sutil.Clientsets, operatorNamespace, cephClusterNamespace, osdID, forceflagValue string) {
+	var answer string
+	logging.Warning("Are you sure you want to purge osd ID %s? If so, enter 'yes'\n", osdID)
+	fmt.Scanf("%s", &answer)
+	if err := PromptToContinueOrCancel(answer, "yes"); err != nil {
+		logging.Fatal(fmt.Errorf("Purge osd osd.%s %s ", osdID, err))
+	}
+
+	if err := cephcommands.CheckPgStatus(ctx, clientsets, operatorNamespace, cephClusterNamespace); err != nil {
```

Comment:
I don't think we need to check PG status. The upstream tool should just check if it's `safe-to-destroy`. And if it's not safe, then we prompt in the upstream tool if the user wants to force it.